apps/molgenis-components/src/components/forms/InputString.vue

```diff
@@ -69,7 +84,10 @@ span:hover .hoverIcon {
     <b>column</b>
     <InputString id="input-string4" :label.sync="column.label" v-model="column.value" :editMeta="true"
                  :description.sync="column.description"/>
-    text.<br/>
+    text.<br/><br/>
+    <InputString id="input-string5" v-model="value" :stringLength="4" label="maximum stringLength (4)"/>
```

Comment:
you are reusing the same value in the different string examples is this by accident or design ?